### PR TITLE
Remove second snowpack instance

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

This gets rid of having 2 snowpack instances. Instead there's just one, but no more `import.meta.env.astro`. So this is a breaking chance.

People can use `import.meta.env.SSR` for the same purpose.

## Testing

- [✅] Tests are passing
- [✅] Tests updated where necessary

## Docs

This wasn't documented, I'm adding it.

### CODE REVIEW

**Comment by matthewp** [comment]
Closes https://github.com/snowpackjs/astro/issues/357

**Comment by FredKSchott** [comment]
Oh my god this just feels so much better! Output looks good and feels faster too.

Saw one issue when running this on snowpack.dev main branch:

```
[build] Error: Error: No renderer found for PluginSearchPage:load! Did you forget to add a renderer to your Astro config?
    at file:///Users/fks/Code/astro/packages/astro/src/build/page.ts:98:44
    at async Promise.all (index 0)
    at buildStaticPage (file:///Users/fks/Code/astro/packages/astro/src/build/page.ts:96:3)
    at file:///Users/fks/Code/astro/packages/astro/src/build.ts:78:9
    at async Promise.all (index 30)
    at build (file:///Users/fks/Code/astro/packages/astro/src/build.ts:75:5)
    at buildAndExit (file:///Users/fks/Code/astro/packages/astro/src/cli.ts:16:15)
```

**Comment by FredKSchott** [comment]
Also HMR is no longer connecting in dev, also on the Snowpack repo (seems to be trying to connect to `localhost:3000`)

**Comment by matthewp** [comment]
@FredKSchott maybe you can show me how you're running a local copy of astro in that repo. I haven't been able to get it to work.

**Comment by matthewp** [comment]
HMR issue is unrelated to this change. Fixed it here: https://github.com/snowpackjs/astro/pull/375

**Comment by matthewp** [comment]
Ok, that took longer that it should have. @FredKSchott reason that component throws is because it depends on the very env variable that this PR removes support for. So changing it to use `import.meta.env.SSR` instead fixes it.

File: packages/astro/src/runtime.ts [hunk 1]
```diff
@@ -439,52 +431,31 @@ async function createSnowpack(astroConfig: AstroConfig, options: CreateSnowpackO
...
+    snowpack,
+    snowpackRuntime,
+    snowpackConfig,
   };
 
   return {
     runtimeConfig,
     load: load.bind(null, runtimeConfig),
-    shutdown: () => Promise.all([backendSnowpack.shutdown(), frontendSnowpack.shutdown()]).then(() => void 0),
+    shutdown: () => Promise.all([snowpack.shutdown()]).then(() => void 0),
```
Comment (drwpow) [inline packages/astro/src/runtime.ts:459]: Maybe remove the `Promise.all()` ? Unless we need to add more cleanup relatively soon

**Review by drwpow — APPROVED** [review]
🎉 🎉 🎉

File: packages/astro/src/runtime.ts (hunk 1 above)
Comment (matthewp) [reply packages/astro/src/runtime.ts:459]: oh yeah good point

**Review by FredKSchott — APPROVED** [review]
LGTM! Plz run it on snowpack.dev if you can to confirm

**Pull request merged**